test/Microsoft.AspNetCore.Authentication.Test/FacebookTests.cs

```diff
@@ -182,10 +181,6 @@ public class FacebookTests
                 app => app.UseAuthentication(),
                 services =>
                 {
-                    services.AddAuthentication(options =>
-                    {
-                        options.DefaultSignInScheme = "External";
-                    });
```

Comment:
@Tratcher @PinpointTownes the benefit of the split is here, we no longer need to specify DefaultSignInScheme when only a single cookie is added, we can leave SignInScheme null on the options, as it just works...